imports/api/transactions/actions.js

```diff
@@ -34,7 +32,7 @@ Transactions.actions = {
       Session.update('modalContext', 'txdef', options.txdef);
       const entity = options.entity;
       const insertTx = Session.get('modalContext').insertTx;
-      doc = _.extend({}, insertTx, doc);
+      doc = _.extend({ communityId: getActiveCommunityId() }, insertTx, doc);
```

Comment:
ha a doc-ban mindenkepp bejon a communityId, akkor az elso parameter az extendbe most mar nem is kell

Reply:
És ha hívásnál adja meg a valaki a docot és nem a defaultDoc lesz az, és abból kimarad a communityId?

Reply:
Akkor az is jo ha csak kicsereljuk az elso parameter a defaulNewDoc-ra